### DISCUSSION TITLE
Countries phone code form field

### DISCUSSION THREAD

**Discussion by Tupakh** [elementor/elementor#22539] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It is annoying not to have this option, where most of the time users omit to write the prefix of the phone number. I tried with multiple plugins but they are not updated on time and are full of bugs.

### Describe the solution you'd like

I want to choose a field type named phone-code or country-code, with/without country flag. To be or not to be stiky to the phone number field.

**Comment by hitboyx7** [comment]
Please, we need this feature for real. It's insane when you have clients from a lot of countries :S We're going crazy :((((

**Comment by hanna-III** [comment]
Voting up. Custom solutions to this are very unreliable
